Processed spikes from trial 100.
Processed spikes from trial 200.
Processed spikes from trial 300.
Processed spikes from trial 100.
Processed spikes from trial 200.
Processed spikes from trial 300.
Traceback (most recent call last):
  File "/project/nicho/caleb/git/intermittent_control_project/code/python_switching_models/HMM_param_search.py", line 125, in <module>
    multifold_shuffled_neuron_order = pd.DataFrame.to_numpy(pd.read_csv(
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 586, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 51, in __init__
    self._open_handles(src, kwds)
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/base_parser.py", line 222, in _open_handles
    self.handles = get_handle(
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/common.py", line 702, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/project/nicho/caleb/git/intermittent_control_project/data/python_switching_models/RSCO_move_window0.05sBins/multifold_neuron_classification.csv'
Traceback (most recent call last):
  File "/project/nicho/caleb/git/intermittent_control_project/code/python_switching_models/HMM_param_search.py", line 125, in <module>
    multifold_shuffled_neuron_order = pd.DataFrame.to_numpy(pd.read_csv(
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 586, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 51, in __init__
    self._open_handles(src, kwds)
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/parsers/base_parser.py", line 222, in _open_handles
    self.handles = get_handle(
  File "/software/python-anaconda-2021.05-el8-x86_64/lib/python3.8/site-packages/pandas/io/common.py", line 702, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/project/nicho/caleb/git/intermittent_control_project/data/python_switching_models/RSCO_move_window0.05sBins/multifold_neuron_classification.csv'
